ccd-adapter/src/main/java/uk/gov/hmcts/cmc/ccd/domain/defendant/CCDRespondent.java

```diff
@@ -81,4 +81,8 @@
     private LocalDateTime redeterminationRequestedDate;
     private CCDMadeBy redeterminationMadeBy;
 
+    private String preferredCourtName;
+    private CCDAddress preferredCourtAddress;
+    private String preferredCourtReason;
```

Comment:
Don't we need these fields in CMC domain as well along with their mappings?

Reply:
There's some partial overlap with DQ (I am not working on that epic) - Sanchita had added an AC in those DQ stories so as to these fields would be mapped/populated as part of saving DQ to the DB. I think @KiranVarma02 is on the case